From ace7cd95dd39bf1c251c4616cef0c408351fde8a Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 00:13:25 +0200
Subject: [PATCH 3/3] Bluetooth: Fix AAPT warning

Fixes:
res/layout/bt_enabling_progress.xml:30: warn:
generated id 'android:id/progress' for external package 'android'.

Change-Id: I356f0e70605d56429892f52f4a74cd0eaf52e3f4
---
 res/layout/bt_enabling_progress.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/layout/bt_enabling_progress.xml b/res/layout/bt_enabling_progress.xml
index 7f2fd72b..fcd73b3c 100644
--- a/res/layout/bt_enabling_progress.xml
+++ b/res/layout/bt_enabling_progress.xml
@@ -27,7 +27,7 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <ProgressBar android:id="@+android:id/progress"
+        <ProgressBar android:id="@android:id/progress"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content" />
 
-- 
2.17.1

